### PR TITLE
[Musl] fix SO_* for MIPS

### PR DESCRIPTION
**Pull request by Cogitri**
Continuation of #2852

### CODE REVIEW

File: src/core/sys/posix/sys/socket.d [hunk 1]
```diff
@@ -1895,23 +1895,49 @@ else version (CRuntime_Musl)
...
-        SO_REUSEPORT    = 15,
-        SO_RCVLOWAT     = 18,
-        SO_SNDLOWAT     = 19,
-        SO_RCVTIMEO     = 20,
-        SO_SNDTIMEO     = 21,
-        SO_ACCEPTCONN   = 30
+        SO_DEBUG        = 1
+    }
+
+    version (MIPS_Any) {
```
Comment (Geod24) [inline src/core/sys/posix/sys/socket.d]: Braces on their own line

**Comment by Cogitri** [comment]
Fixed, thanks

File: src/core/sys/posix/sys/socket.d [hunk 2]
```diff
@@ -1895,23 +1895,50 @@ else version (CRuntime_Musl)
...
+            SO_OOBINLINE    = 0x0100,
+            SO_LINGER       = 0x0080,
+            SO_REUSEPORT    = 0x0200,
+            SO_RCVLOWAT     = 0x1004,
+            SO_SNDLOWAT     = 0x1003,
+            SO_RCVTIMEO     = 0x1006,
+            SO_SNDTIMEO     = 0x1005,
+            SO_ACCEPTCONN   = 0x1009
+        }
+    } else {
```
Comment (Geod24) [inline src/core/sys/posix/sys/socket.d]: Actually, you forgot this

Comment (Cogitri) [reply src/core/sys/posix/sys/socket.d]: Oh right, thanks, I guess this isn't quite my style :P

**Pull request merged**